playgrounds/minimal-react/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "minimal-react",
+  "version": "1.0.0",
+  "main": "index.js",
+  "private": true,
+  "license": "private",
+  "scripts": {
+    "start": "webpack-dev-server"
+  },
+  "dependencies": {
+    "@babel/core": "^7.11.1",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@emotion/babel-preset-css-prop": "^10.0.27",
+    "@emotion/core": "^10.0.34",
+    "react": "17.0.0-rc.0",
+    "react-dom": "17.0.0-rc.0"
+  },
+  "devDependencies": {
+    "babel-loader": "^8.1.0",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0"
+  },
+  "eslintConfig": {
+    "rules": {
+      "react/react-in-jsx-scope": "off"
+    }
+  },
+  "babel": {
+    "presets": [
+      "@babel/preset-env",
+      [
+        "@babel/preset-react",
+        {
+          "runtime": "automatic"
```

Comment:
oh: I think this is unnecessary to configure here if babel goes after, right? 

In any case, I'd guess users should probably have a shared `reactJsxOptions` config object they use for both in babel.config.js that they use for both. Something about consistency.